src/accounts/pages/CallBack.tsx

```diff
@@ -83,8 +83,11 @@ export default function CallBackPage() {
         (e.code === SdkErrors.UserLogout || e.code === SdkErrors.UserCancelled)
       ) {
         try {
-          const { requests } = await getLoginRequestFromUrl();
-          const managedRequests = new RequestsManager(requests);
+          const { error } = getLoginRequestResponseFromUrl();
+
+          if (!error) throw new Error("Error not defined");
+          const managedRequests = new RequestsManager(error.requests);
```

Comment:
we are getting requests from error?

Reply:
I have tried to standardise our response format:

```ts
{
  success: boolean
  error: object
  response: object
}
```

This is a standard structure we can use for all request responnses, if there is an error error object will be populated, and if there is not an error the response object will be populated. This lets us easily manage error handling.

In this case there is an error, and because of the semantics described above, the responses are sent as part of the error object.